exercises/strings/strings1.js

```diff
@@ -0,0 +1 @@
+// Add exercise here
```

Comment:
rename this file to string_quiz_1.js?

Reply:
I don't think `quiz` should be in the filename. It's already under the `exercises` folder which should suffice?

Reply:
Okk